src/features/NodeBalancers/NodeBalancerConfigPanel.tsx

```
@@ -356,6 +405,80 @@ class NodeBalancerConfigPanel extends React.Component<CombinedProps> {
               <Divider className={classes.divider} />
             </Grid>
           </Grid>
+
+          <Grid container>
+            <Grid item xs={12}>
+              <Typography variant="title">Choose Backend IPs</Typography>
+            </Grid>
+            <Grid item xs={12}>
+              {
+                nodes.map((node, idx) => {
+                  const hasErrorFor = getAPIErrorFor({
+                    label: 'label',
+                    address: 'address',
+                    weight: 'weight',
+                    mode: 'mode',
+                  }, filterErrors(idx)(errors || []));
+
+                  return (
+                    <Grid key={idx} container>
+                      <Grid item xs={12} md={3}>
+                        <TextField
+                          label="Label"
+                          value={node.label}
+                          onChange={e => onNodeLabelChange(idx, e.target.value)}
+                          errorText={hasErrorFor('label')}
+                        />
+                      </Grid>
+                      <Grid item xs={12} md={3}>
+                        <TextField
+                          label="Address"
+                          value={node.address}
+                          onChange={e => onNodeAddressChange(idx, e.target.value)}
+                          errorText={hasErrorFor('address')}
+                        />
+                      </Grid>
+                      <Grid item xs={12} md={3}>
+                        <TextField
+                          label="Weight"
+                          type="number"
+                          value={node.weight}
+                          onChange={e => onNodeWeightChange(idx, e.target.valueAsNumber)}
```

The value is typed as a number, but upon clearing the field, you get NaN and this error is thrown

```
Warning: Received NaN for the `value` attribute. If this is expected, cast the value to a string.
```

We'll either need to default to a placeholder number when the user clears the field or allow an empty string